woodwork/tests/type_system/test_custom_types.py

```diff
@@ -57,7 +57,7 @@ def new_double_func(series):
     )
     ww.type_system.update_inference_function("Integer", inference_function=None)
     sample_df.ww.init()
-    assert isinstance(sample_df.ww["age"].ww.logical_type, Double)
-    assert sample_df["age"].dtype == "float64"
+    assert isinstance(sample_df.ww["age"].ww.logical_type, IntegerNullable)
```

Comment:
I think this test is broken. I believe the original intent of this test was to confirm that we can replace the default inference function and that it gets used as expected. Before we were replacing the Double inference function and checking that the logical type was identified as Double. Now we are replacing the Double function, but asserting that the type is IntegerNullable.

There might be a better way to go about this, but maybe think through this and make sure the test is still testing what we want.

Reply:
Great point, I updated the test to override the default `IntegerNullable` function instead and check that the logical type is identified as `Double`.